test/database_manager/test_database_manager.py

```diff
@@ -152,8 +152,9 @@ def test_get_inference_deployments_raises_sqlalchemy_error(db_manager: DatabaseM
 def test_get_iqe_record(db_manager, database_reset):
     image_query: ImageQuery = create_iqe(
         detector_id=prefixed_ksuid("det_"),
-        result_type=ResultTypeEnum.binary_classification,
-        label="test_label",
+        mode=ModeEnum.BINARY,
```

Comment:
Can you add a test for creating a Count-mode IQE record as well?

Reply:
Just to make sure the DB is compatible with this change

Reply:
Good idea, will do